1_cpp_basics/readme.md

```diff
@@ -0,0 +1,572 @@
+#                    **Basics Of C++**
+
+
+- ## **What are Arrays :**
```

Comment:
Can be treated as Heading/Sub-Headings. No need to add them under bulletin points.